Docs/ProjectSpec.md

```diff
@@ -838,6 +839,19 @@ schemes:
       customArchiveName: MyTarget
       revealArchiveInOrganizer: false
 ```
+### Test Plan
+
+- [x] **path**: **String** - path that provides the `xctestplan` location.
+- [ ] **defaultPlan**: **Bool** - a bool that defines if given plan is the default one. Defaults to false.
```

Comment:
here we call it `defaultPlan`. In  [Line 853](https://github.com/yonaskolb/XcodeGen/pull/846/files#diff-938b5cf412212f1137a53f072512963aR853) we name it  `default`

Reply:
Changed in 642ba14